src/run/parser/livesplit.rs

```diff
@@ -211,7 +211,7 @@ fn parse_segment<R: BufRead>(
                                 *segment.comparison_mut(&comparison) = t;
                             })?;
                         }
-                        run.add_custom_comparison(comparison);
+                        run.add_custom_comparison(comparison).map_err(|_| {});
```

Comment:
Yeah, this definitely needs to be thrown. But I see why you can't: Because we add the same comparisons all the time. So you should ignore that certain error case for the livesplit parser (i.e. only throw the Race error). So that would be:
```rust
if let Err(ComparisonError::NameStartsWithRace) = run.add_comparison_name(comparison) { 
    return ... 
}
```